.travis.yml

```diff
@@ -5,9 +5,9 @@ python:
 - "2.7"
 - "3.7"
 install:
-- pip install pyyaml
+- pip install pyyaml Jinja2
```

Comment:
we can do `pip install .` instead, since now we have the `setup.py` file as part of the repo